src/main/java/com/google/sps/servlets/FindLocationServlet.java

```diff
@@ -0,0 +1,24 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/find-location")
+public class FindLocationServlet extends HttpServlet {
+
+ /**
+  * Given the correlator, this will update the database to reflect that the user clicked 'fiind nearby location'.
```

Comment:
Either in this description, or in the param comment if you choose to leave it in, you should actually specify the format the correlator should come it. So whats the query param its attached to? Is it a string? Number? boolean?

Reply:
Done